robot_smach_states/src/robot_smach_states/reset.py

```diff
@@ -89,6 +89,6 @@ def __init__(self,robot, timeout=0.0):
 
     def execute(self, userdata=None):
         self.robot.reset_all_arms()
-        self.robot.head.reset(timeout=self.timeout)
+        self.robot.head.reset()
```

Comment:
Why?

Reply:
reset no longer takes a timeout parameter. The head selfreset function is fixed by
[6ad5f0d](https://github.com/tue-robotics/tue_robocup/pull/1226/commits/6ad5f0d1e77b747098c2162b53c2c95ef79e5c1f)